proofs/signatures/th_lira.plf

```diff
@@ -52,3 +52,21 @@
     ((/_Real x y) (/_Real (reify_real_term x) (reify_real_term y)))
   ))
 
+; Predicates of the form (term Integer) >= (term Real) or negations thereof
+(declare >=_IntReal (! x (term Int) (! y (term Real) formula)))
```

Comment:
Given that we have conversions above, we would we need this predicate? Can't we use the above conversions and then use >=_Real?

Reply:
Good question. Sometimes we need to know that one side of the inequality is integral so that we can tighten the bound (e.g. go from `x < 4.5` to `x <= 4`). If we don't have a theorem of the form "the (integral) left side is less than the right", its not clear to to write a sound tightening axiom.

Reply:
Thanks for the explanation, makes sense.